doc/source/MANUAL.rst

```diff
@@ -103,6 +105,22 @@ specified and pointing to an existing file or the default location
 be ignored. Configuration embedded in a ``tox.ini`` will only be used if other
 configuration files are not present.
 
+pyproject.toml
+''''''''''''''
+
+Similarly, if your project is using ``pyproject.toml``, you may forego the
+config file, and instead create a ``[tool.stestr]`` section with the desired
+configuration options.  For example::
```

Comment:
I think it'll be good to document the priorty order between the `.stestr.conf`, `tox.ini`, and `pyproject.toml`. While I think it's unlikely in practice for someone to configure multiple in a repository being explicit here about how this will be loaded under the covers and making that semi-official will likely prevent issues down the road in case someone does.

Reply:
Can do.